<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1162</width>
    <height>624</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>170</y>
      <width>691</width>
      <height>401</height>
     </rect>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QTabWidget" name="tabWidget_2">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>170</y>
      <width>251</width>
      <height>191</height>
     </rect>
    </property>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QTabWidget" name="tabWidget_3">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>370</y>
      <width>251</width>
      <height>201</height>
     </rect>
    </property>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_6">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QTabWidget" name="tabWidget_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>181</width>
      <height>391</height>
     </rect>
    </property>
    <widget class="QWidget" name="tab_7">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_8">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QTabWidget" name="tabWidget_5">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1151</width>
      <height>171</height>
     </rect>
    </property>
    <widget class="QWidget" name="tab_9">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_file">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>201</width>
        <height>121</height>
       </rect>
      </property>
      <property name="title">
       <string>FILE</string>
      </property>
      <widget class="QPushButton" name="pushButton_addImages">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>10</y>
         <width>81</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>Add Images</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_addImages">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>60</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_zoom">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>19</y>
        <width>120</width>
        <height>111</height>
       </rect>
      </property>
      <property name="title">
       <string>ZOOM</string>
      </property>
      <widget class="QPushButton" name="pushButton_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Zoom In</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Zoom Out</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Pan</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_layout">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>30</y>
        <width>120</width>
        <height>80</height>
       </rect>
      </property>
      <property name="title">
       <string>LAYOUT</string>
      </property>
      <widget class="QPushButton" name="pushButton_defaultLoyout">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>75</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_cameraModel_2">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>20</y>
        <width>201</width>
        <height>111</height>
       </rect>
      </property>
      <property name="title">
       <string>CAMERA MODEL</string>
      </property>
      <widget class="QGroupBox" name="groupBox_cameraModel">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>20</y>
         <width>115</width>
         <height>79</height>
        </rect>
       </property>
       <property name="title">
        <string>Camera Model</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QRadioButton" name="radioButton_standard">
          <property name="text">
           <string>Standard</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_fisheye">
          <property name="text">
           <string>Fisheye</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QPushButton" name="pushButton_cameraModelOptions">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>30</y>
         <width>61</width>
         <height>61</height>
        </rect>
       </property>
       <property name="text">
        <string>Options</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_calibrate">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>20</y>
        <width>120</width>
        <height>101</height>
       </rect>
      </property>
      <property name="title">
       <string>CALIBRATE</string>
      </property>
      <widget class="QPushButton" name="pushButton_calibrate">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>20</y>
         <width>61</width>
         <height>61</height>
        </rect>
       </property>
       <property name="text">
        <string>Calibrate</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_view">
      <property name="geometry">
       <rect>
        <x>830</x>
        <y>19</y>
        <width>181</width>
        <height>101</height>
       </rect>
      </property>
      <property name="title">
       <string>VIEW</string>
      </property>
      <widget class="QPushButton" name="pushButton_showUndistorted">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>131</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Show Undistorted</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Fisheye Scale</string>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="doubleSpinBox">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>60</y>
         <width>62</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_export">
      <property name="geometry">
       <rect>
        <x>1020</x>
        <y>20</y>
        <width>120</width>
        <height>101</height>
       </rect>
      </property>
      <property name="title">
       <string>EXPORT</string>
      </property>
      <widget class="QPushButton" name="pushButton_export">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Export</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_10">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1162</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
